packages/cxl-ui/scss/cxl-minidegree-header.scss

```diff
@@ -0,0 +1,116 @@
+@use "~@conversionxl/cxl-lumo-styles/scss/mq";
+@use "./mixins";
+
+:host {
+  color: var(--lumo-tint);
+  background-color: var(--lumo-shade);
+
+  @media #{mq.$medium} {
+    @include mixins.pesudo-element-full-width(var(--lumo-shade));
+  }
+
+  .container {
+    position: relative;
+    box-sizing: border-box;
+    width: 100%;
+    max-width: var(--cxl-content-max-width-wide);
+    padding: var(--lumo-space-l) 0;
+
+    @media #{mq.$small} {
+      display: flex;
+      justify-content: space-between;
+      gap: var(--lumo-space-m);
+    }
+
+    @media #{mq.$large} {
+      padding: var(--lumo-space-xl) 0;
+    }
+  }
+
+  .content {
+    max-width: 685px;
```

Comment:
Where does it come from? Is it used anywhere else? Maybe worth making global var?

Reply:
@pawelkmpt Not used elsewhere. I can also make it percentage-based, for example. Or the variable could be placed at the beginning of the same component. What do you think about these solutions?

Reply:
I recall @lkraav mentioning max-width of content somewhere, so I thought maybe it's this value. What is set in featured course slider? It also has width limit.

Reply:
`max-width: 650px;`

I can replace those values with a global variable. Would `--cxl-hero-content-max-width` be okay?